ContentfulDeliveryAPI/CDAConfiguration.h

```diff
@@ -16,7 +16,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-extern const NSString* const CDA_DEFAULT_SERVER;
+extern NSString* const CDA_DEFAULT_SERVER;
```

Comment:
This was a compile error  before and it had to be changed. I'm not sure I understand your questions though. AFAIK, it isnt' writable.